### DISCUSSION TITLE
How to  restore  index

### DISCUSSION THREAD

**Discussion by 1292253144** [milvus-io/milvus#27560] [Q&A and General discussion]
I found that the data I backed up could not recover the index.  milvus version is 2.2.8

**Comment by yhmo** [comment]
The backup tool doesn't restore index, this behavior is as-designed.
The reason is: after the data is imported, the data is re-organized to new segments. As we know, milvus builds an independent index for each segment, once the segment is rebuilt, the index also need to be rebuilt.
So, after restore, you need to manually create the index.
